src/ui/sass/modules/_SearchBar.scss

```diff
@@ -121,6 +122,7 @@ $searchbar-button-text-color-active: var(--yxt-searchbar-button-text-color-base)
   &-clear
   {
     display: flex;
+    flex: 0 0 auto;
```

Comment:
would just setting flex-shrink: 0 here work? for my edification why do we need flex-shrink: 0 here?

Reply:
You right, all we need is flex-shrink: 0. It prevents the buttons from shrinking. I used the full flex syntax because a  guide recommended using that syntax, however I can switch